### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/695c0f882ff15365a803b0efcce9666c002fd3f5

### CODE REVIEW

File: en/firefox/browsers/compare/safari.ftl [hunk 1]
```diff
@@ -40,11 +40,17 @@ compare-safari-in-addition-to-the-regular = In addition to the regular set of fe
 #   $attrs (string) - link to addons.mozilla.org with additional attributes for analytics
 compare-safari-like-safari-firefox-encourages = Like { -brand-name-safari }, { -brand-name-firefox } encourages its enthusiastic developer community to create <a { $attrs }>add-ons and extensions</a> to the browser. And since our platform is open-source, there’s a vast selection adding a wealth of functionality.
 
+# Variables:
+#   $pocket (string) - link to getpocket.com with additional attributes for analytics
+compare-safari-also-when-you-sign-up-for-updated = Also, when you sign up for a { -brand-name-firefox } account you get access to unique services like <a { $pocket }>{ -brand-name-pocket }</a> that integrate directly into the browser. The { -brand-name-pocket } for { -brand-name-firefox } button lets you save web pages and videos to { -brand-name-pocket } in just one click, so you can read a clean, distraction-free version whenever and wherever you want — even offline.
```
Comment (peiying2) [inline en/firefox/browsers/compare/safari.ftl:45]: There should be a comma after the clause so it looks like `Also, when.... account, you get...`

**Review by peiying2 — CHANGES_REQUESTED** [review]
@craigcook Punctuation change request.

File: en/firefox/browsers/compare/safari.ftl (hunk 1 above)
Comment (craigcook) [reply en/firefox/browsers/compare/safari.ftl:45]: Fixed!

**Comment by alexgibson** [comment]
Closing in favour of https://github.com/mozilla-l10n/www-l10n/pull/109

**Pull request closed without merging**